keanu-python/keanu/tensor.py

```diff
@@ -72,8 +72,9 @@ def __get_tensor_from_scalar(scalar: primitive_types) -> JavaObject:
             raise NotImplementedError("Generic types in a ndarray are not supported. Was given {}".format(type(scalar)))
 
     @staticmethod
-    def _to_ndarray(java_tensor: JavaObject) -> numpy_types:
+    def _to_ndarray(java_tensor: JavaObject, return_as_primitive: bool = False) -> Any:
```

Comment:
Maybe we should make the default `True` since we are using this more in `sampling` than other places

Reply:
Ok